src/rules.rs

```diff
@@ -66,6 +67,7 @@ impl Default for Rules {
             min_word_count: 1,
             max_word_count: 14,
             min_characters: 0,
+            max_characters: std::usize::MAX,
```

Comment:
`std::usize::MAX` is deprecated, could you replace it with `usize::MAX`? https://doc.rust-lang.org/std/primitive.usize.html#associatedconstant.MAX

Reply:
Yep just saw that and already replaced it everywhere.

Reply:
Done...